islandora_solr_config/includes/atom_results.inc

```diff
@@ -0,0 +1,248 @@
+<?php
+
+/**
+ * @file
+ * Contains methods to search solr and display results.  depends on
+ * Apache_Solr_Php client.
+ */
+
+/**
+ * Extention of IslandoraSolrResults for templating purposes.
+ * This overrides the displayResults function to provide an alternate display
+ * type.
+ */
+
+class IslandoraSolrResultsAtom extends IslandoraSolrResults {
+
+  /**
+   * Outputs results basically in the normal way.
+   *
+   * Thumbnails pulled from the Fedora repository.
+   *
+   * @param object $islandora_solr_query
+   *   A solr query object.
+   *
+   * @return string
+   *   html output for the resultset. Note: we currently create this
+   *   output manually, should refactor to use drupal forms api.
+   */
+  public function printAtom($islandora_solr_query, $title = "Search Results") {
+
+    global $base_url;
+
+    drupal_add_http_header('Content-Type', 'application/atom+xml; charset=utf-8');
+    $islandora_solr_query->solrLimit = variable_get('islandora_solr_config_rss_limit', 50);
```

Comment:
Should we use the RSS limit for atom? Maybe it should have its own limit. Or the limit should specify its for both RSS and ATOM.

Reply:
I don't think it needs two separate limits; the feeds accomplish the same thing. Better to tweak the limit description to say it works for both.